pom.xml

```diff
@@ -167,6 +168,7 @@
 
     <build>
         <plugins>
+<!--
             <plugin>
                 <groupId>org.sonatype.plugins</groupId>
```

Comment:
My wife just gave birth (yay!), just letting you know I will need another week before I can look into this.  Thanks for your patience.